healthcheck.js

```diff
@@ -0,0 +1,153 @@
+"use strict";
+
+const os = require('os');
+const pjson = require('./package.json');
+const config = require('./config');
+const HealthChecker = require('@dojot/healthcheck').HealthChecker;
+const DataTrigger = require('@dojot/healthcheck').DataTrigger;
+const logger = require("@dojot/dojot-module-logger").logger;
+
+// cron Manager
+var cronManager = null; // initialized at init()
+
+//health checker
+const healthCheckerConfig = {
+    description: 'health of cron service',
+    version: pjson.version,
+    status: 'pass'
+};
+
+const healthChecker = new HealthChecker(healthCheckerConfig);
+
+// uptime
+const uptime = {
+    measurementName: 'uptime',
+    componentType: 'system',
+    observedUnit: 's',
+    status: 'pass'
+};
+
+const uptimeCollector = (trigger = DataTrigger) => {
+    let value = Math.floor(process.uptime());
+    trigger.trigger(value, 'pass');
+    logger.debug(`[HEALTH CHECK] SYSTEM UPTIME ${value}`);
```

Comment:
I don't think that adding logs in health check is advisable. We could end up with lots of "Cheking memory"-like output lines which would remove useful logs.

Reply:
Ok. It makes sense.